Makefile

```diff
@@ -81,7 +81,7 @@ PULL_POLICY ?= Always
 # Set build time variables including version details
 LDFLAGS := $(shell ./hack/version.sh)
 
-KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.30.0
+KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.32.0
```

Comment:
```suggestion
KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.30.0
```

Reply:
ref - [https://storage.googleapis.com/kubebuilder-tools/](https://storage.googleapis.com/kubebuilder-tools/)

Reply:
@Amulyam24 ^^^

Reply:
any idea why it is only published till 1.30? need to check if they are plublishing elsewhere.

Reply:
@Prajyot-Parab, we don't need the script `fetch_ext_bins.sh` as envtest will handle installing the binaries.